pages/posts/@id/+onBeforeRender.ts

```diff
@@ -8,7 +8,8 @@ export default async function onBeforeRender(pageContext: PageContextBuiltInServ
     const knownQueries = useQueriesState.getState().knownQueries
     const { routeParams: { id } } = pageContext
 
-    if (knownQueries.has(hashKey(postsQueries.detail(id).queryKey).valueOf()) === false) {
+    const queryKey = hashKey(postsQueries.detail(id).queryKey).valueOf()
+    if (knownQueries.has(queryKey) === false) {
```

Comment:
I will change this to `if(!knownQueries.get(queryKey))`, cause we also need to check if the value is false.

[Map & Set](https://javascript.info/map-set) : 
- map.get(key) – returns the value by the key, `undefined` if key doesn’t exist in map.
- map.has(key) – returns true if the key exists, false otherwise.